recaptcha/src/delete_key.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/*
+ * Copyright 2020 Google LLC.
```

Comment:
nit: the new files should all use 2021 as the copyright year
```suggestion
 * Copyright 2021 Google LLC.
```